core/crypto/vrf/p256/p256.go

```diff
@@ -37,9 +37,10 @@ import (
 	"fmt"
 	"math/big"
 
-	"github.com/golang/protobuf/proto"
 	"github.com/google/keytransparency/core/crypto/vrf"
 	"github.com/google/trillian/crypto/keys"
+
```

Comment:
I think we should come up with a strategy to group imports :)

Reply:
This follows the following import strategy:
1. golang imports
2. KT and Trillian imports
3. Third party imports
4. Underscore and renamed imports